src/objects/Hierarchy_Settings__c.object

```diff
@@ -57,6 +67,16 @@
         <type>Checkbox</type>
     </fields>
     <fields>
+        <fullName>Contacts_Addresses_Enabled__c</fullName>
```

Comment:
Because we don't use Contact record types in HEDA. Account Addresses will be enabled for those record types that are selected. Probably seeing the UI will help: 

![screen shot 2016-08-15 at 3 51 50 pm](https://cloud.githubusercontent.com/assets/1894238/17707223/05376c16-63a5-11e6-87df-f5241e67041f.png)